File: app/src/main/java/com/tt/handsomeman/service/SignUpAddPayoutService.java
package com.tt.handsomeman.service;

import com.tt.handsomeman.model.UserActivatingAccount;
import com.tt.handsomeman.response.StandardResponse;
import com.tt.handsomeman.util.Constants;

import retrofit2.Call;
import retrofit2.http.Body;
import retrofit2.http.Header;
import retrofit2.http.POST;
import retrofit2.http.Query;

public interface SignUpAddPayoutService {
    @POST(Constants.SIGN_UP_ADD_PAYOUT_SUFFIX)
    Call<StandardResponse> doSignUpAddPayout(@Header("Authorization") String token, @Body UserActivatingAccount userActivatingAccount, @Query("kindOfHandyman") String kindOfHandyman);
}

File: app/src/main/java/com/tt/handsomeman/service/SignUpAddPayoutService.kt
package com.tt.handsomeman.service

import com.tt.handsomeman.model.UserActivatingAccount
import com.tt.handsomeman.response.StandardResponse
import com.tt.handsomeman.util.Constants

import retrofit2.Call
import retrofit2.http.Body
import retrofit2.http.Header
import retrofit2.http.POST
import retrofit2.http.Query

interface SignUpAddPayoutService {
    @POST(Constants.SIGN_UP_ADD_PAYOUT_SUFFIX)
    fun doSignUpAddPayout(@Header("Authorization") token: String, @Body userActivatingAccount: UserActivatingAccount, @Query("kindOfHandyman") kindOfHandyman: String): Call<StandardResponse>
}
